javascript/sbp/RELEASE-VERSION

```diff
@@ -1 +1 @@
-4.5.0
\ No newline at end of file
+4.5.1-alpha
```

Comment:
`package.json` and `package.lock` seem to be missing, there's a bug in javascript here that it sometimes misses these, you should run `make javascript` and see if it fixes this